schema/swagger.yml

```diff
@@ -66,9 +70,9 @@ paths:
       - Accounts
       summary: Requests account data
       operationId: getAccounts
-      description:
-        By passing in the appropriate parameters, you can search for
-        matching accounts in the system
+      description: |
+        By passing the appropriate parameters, you can search for matching accounts in the system.
+        Otherwise returns a list of accounts.
```

Comment:
I believe we need to rephrase such descriptions, e.g. `By passing the appropriate parameters`  is really not needed to mention. Its something obvious, instead we put more focus on adding more details to parameters documentation.

Reply:
I simplified the descriptions now